ui/static/ui/js/listing.js

```diff
@@ -213,6 +229,72 @@ define(
         });
       });
 
+      var updateCheckDisplay = function(node) {
+        if ($(node).data("selected")) {
+          $(node).find("i").attr("class", "fa fa-check");
+        } else {
+          $(node).find("i").attr("class", "fa fa-arrow-right");
+        }
+      };
+      /**
+       * Update export count badge
+       * @param {number} direction Add this value to
+       * the existing count before rendering
+       */
+      var updateExportCount = function(direction) {
+        var oldCount = $("#export_button").data("export-count");
+        var newCount = oldCount + direction;
+
+        $("#export_button .badge").remove();
```

Comment:
I was avoiding React here for consistency with the rest of the listing page.

`errorText` and `messageText` in `lr_exports.jsx` are text strings and don't manipulate the DOM, they're escaped and included as text. Is that what you're referring to? The line numbers may have slipped since last push